.github/create-release.sh

```diff
@@ -2,14 +2,7 @@
 
 # Remove old 'build' folder
 if [ -d build ]; then
-  read -p "Will delete 'build/' folder. Is that OK? [y/N] " -r
-
-  if [[ ! $REPLY =~ ^[Yy]$ ]]
-  then
-    exit 1
-  else
-    rm -rf build
-  fi
+  rm -rf build
```

Comment:
While yes, it is questionable whether this check is worth it, instead of removing it, you can simply run `echo 'y' | ./create-release.sh` in github action instead.